### DISCUSSION TITLE
This site can’t be reached

### DISCUSSION THREAD

**Discussion by paivadaniel** [photoprism/photoprism#3378] [Q&A]
Hello, I'm using docker-compose.yml available at Get Started on Photoprism's official website, after running:

sudo docker-compose up -d

When accessing 0000:2342, the following error is occurring:

This site can’t be reached
The webpage at http://0.0.0.0:2342/ might be temporarily down or it may have moved permanently to a new web address.
ERR_SOCKET_NOT_CONNECTED

Following is the output of: sudo docker-compose logs --tail=100

Attaching to photoprism2_photoprism_1, photoprism2_mariadb_1
mariadb_1     | 2023-05-06 02:38:21+00:00 [Note] [Entrypoint]: Entrypoint script for MariaDB Server 1:10.11.2+maria~ubu2204 started.
mariadb_1     | 2023-05-06 02:38:21+00:00 [Note] [Entrypoint]: Switching to dedicated user 'mysql'
mariadb_1     | 2023-05-06 02:38:21+00:00 [Note] [Entrypoint]: Entrypoint script for MariaDB Server 1:10.11.2+maria~ubu2204 started.
mariadb_1     | 2023-05-06 02:38:22+00:00 [Note] [Entrypoint]: Initializing database files
mariadb_1     | 
mariadb_1     | 
mariadb_1     | PLEASE REMEMBER TO SET A PASSWORD FOR THE MariaDB root USER !
mariadb_1     | To do so, start the server, then issue the following command:
mariadb_1     | 
mariadb_1     | '/usr/bin/mariadb-secure-installation'
mariadb_1     | 
mariadb_1     | which will also give you the option of removing the test
mariadb_1     | databases and anonymous user created by default.  This is
mariadb_1     | strongly recommended for production servers.
mariadb_1     | 
mariadb_1     | See the MariaDB Knowledgebase at https://mariadb.com/kb
mariadb_1     | 
mariadb_1     | Please report any problems at https://mariadb.org/jira
mariadb_1     | 
mariadb_1     | The latest information about MariaDB is available at https://mariadb.org/.
mariadb_1     | 
mariadb_1     | Consider joining MariaDB's strong and vibrant community:
mariadb_1     | https://mariadb.org/get-involved/
mariadb_1     | 
mariadb_1     | 2023-05-06 02:38:49+00:00 [Note] [Entrypoint]: Database files initialized
mariadb_1     | 2023-05-06 02:38:49+00:00 [Note] [Entrypoint]: Starting temporary server
mariadb_1     | 2023-05-06 02:38:49+00:00 [Note] [Entrypoint]: Waiting for server startup
mariadb_1     | 2023-05-06  2:38:49 0 [Note] Starting MariaDB 10.11.2-MariaDB-1:10.11.2+maria~ubu2204 source revision cafba8761af55ae16cc69c9b53a341340a845b36 as process 96
mariadb_1     | 2023-05-06  2:38:50 0 [Note] InnoDB: Compressed tables use zlib 1.2.11
mariadb_1     | 2023-05-06  2:38:50 0 [Note] InnoDB: Number of transaction pools: 1
mariadb_1     | 2023-05-06  2:38:50 0 [Note] InnoDB: Using crc32 + pclmulqdq instructions
mariadb_1     | 2023-05-06  2:38:50 0 [Note] mysqld: O_TMPFILE is not supported on /tmp (disabling future attempts)
mariadb_1     | 2023-05-06  2:38:50 0 [Note] InnoDB: Using liburing
mariadb_1     | 2023-05-06  2:38:50 0 [Note] InnoDB: Initializing buffer pool, total size = 512.000MiB, chunk size = 8.000MiB
mariadb_1     | 2023-05-06  2:38:50 0 [Note] InnoDB: Completed initialization of buffer pool
mariadb_1     | 2023-05-06  2:38:50 0 [Note] InnoDB: File system buffers for log disabled (block size=4096 bytes)
mariadb_1     | 2023-05-06  2:38:50 0 [Note] InnoDB: 128 rollback segments are active.
mariadb_1     | 2023-05-06  2:38:50 0 [Note] InnoDB: Setting file './ibtmp1' size to 12.000MiB. Physically writing the file full; Please wait ...
mariadb_1     | 2023-05-06  2:38:50 0 [Note] InnoDB: File './ibtmp1' size is now 12.000MiB.
mariadb_1     | 2023-05-06  2:38:50 0 [Note] InnoDB: log sequence number 46456; transaction id 14
mariadb_1     | 2023-05-06  2:38:50 0 [Note] Plugin 'FEEDBACK' is disabled.
mariadb_1     | 2023-05-06  2:38:50 0 [Warning] 'user' entry 'root@bc8abba794c5' ignored in --skip-name-resolve mode.
mariadb_1     | 2023-05-06  2:38:50 0 [Warning] 'proxies_priv' entry '@% root@bc8abba794c5' ignored in --skip-name-resolve mode.
mariadb_1     | 2023-05-06  2:38:50 0 [Note] mysqld: ready for connections.
mariadb_1     | Version: '10.11.2-MariaDB-1:10.11.2+maria~ubu2204'  socket: '/run/mysqld/mysqld.sock'  port: 0  mariadb.org binary distribution
mariadb_1     | 2023-05-06 02:38:50+00:00 [Note] [Entrypoint]: Temporary server started.
mariadb_1     | 2023-05-06 02:38:50+00:00 [Note] [Entrypoint]: Creating database photoprism
mariadb_1     | 2023-05-06 02:38:50+00:00 [Note] [Entrypoint]: Creating user photoprism
mariadb_1     | 2023-05-06 02:38:50+00:00 [Note] [Entrypoint]: Giving user photoprism access to schema photoprism
mariadb_1     | 2023-05-06 02:38:50+00:00 [Note] [Entrypoint]: Securing system users (equivalent to running mysql_secure_installation)
mariadb_1     | 
mariadb_1     | 2023-05-06 02:38:50+00:00 [Note] [Entrypoint]: Stopping temporary server
mariadb_1     | 2023-05-06  2:38:50 0 [Note] mysqld (initiated by: unknown): Normal shutdown
mariadb_1     | 2023-05-06  2:38:50 0 [Note] InnoDB: FTS optimize thread exiting.
mariadb_1     | 2023-05-06  2:38:51 0 [Note] InnoDB: Starting shutdown...
mariadb_1     | 2023-05-06  2:38:51 0 [Note] InnoDB: Dumping buffer pool(s) to /var/lib/mysql/ib_buffer_pool
mariadb_1     | 2023-05-06  2:38:51 0 [Note] InnoDB: Buffer pool(s) dump completed at 230506  2:38:51
mariadb_1     | 2023-05-06  2:38:51 0 [Note] InnoDB: Removed temporary tablespace data file: "./ibtmp1"
mariadb_1     | 2023-05-06  2:38:51 0 [Note] InnoDB: Shutdown completed; log sequence number 46456; transaction id 15
mariadb_1     | 2023-05-06  2:38:51 0 [Note] mysqld: Shutdown complete
mariadb_1     | 
mariadb_1     | 2023-05-06 02:38:51+00:00 [Note] [Entrypoint]: Temporary server stopped
mariadb_1     | 
mariadb_1     | 2023-05-06 02:38:51+00:00 [Note] [Entrypoint]: MariaDB init process done. Ready for start up.
mariadb_1     | 
mariadb_1     | 2023-05-06  2:38:51 0 [Note] Starting MariaDB 10.11.2-MariaDB-1:10.11.2+maria~ubu2204 source revision cafba8761af55ae16cc69c9b53a341340a845b36 as process 1
mariadb_1     | 2023-05-06  2:38:51 0 [Note] InnoDB: Compressed tables use zlib 1.2.11
mariadb_1     | 2023-05-06  2:38:51 0 [Note] InnoDB: Number of transaction pools: 1
mariadb_1     | 2023-05-06  2:38:51 0 [Note] InnoDB: Using crc32 + pclmulqdq instructions
mariadb_1     | 2023-05-06  2:38:51 0 [Note] mysqld: O_TMPFILE is not supported on /tmp (disabling future attempts)
mariadb_1     | 2023-05-06  2:38:51 0 [Note] InnoDB: Using liburing
mariadb_1     | 2023-05-06  2:38:51 0 [Note] InnoDB: Initializing buffer pool, total size = 512.000MiB, chunk size = 8.000MiB
mariadb_1     | 2023-05-06  2:38:51 0 [Note] InnoDB: Completed initialization of buffer pool
mariadb_1     | 2023-05-06  2:38:51 0 [Note] InnoDB: File system buffers for log disabled (block size=4096 bytes)
mariadb_1     | 2023-05-06  2:38:51 0 [Note] InnoDB: 128 rollback segments are active.
mariadb_1     | 2023-05-06  2:38:51 0 [Note] InnoDB: Setting file './ibtmp1' size to 12.000MiB. Physically writing the file full; Please wait ...
mariadb_1     | 2023-05-06  2:38:51 0 [Note] InnoDB: File './ibtmp1' size is now 12.000MiB.
mariadb_1     | 2023-05-06  2:38:51 0 [Note] InnoDB: log sequence number 46456; transaction id 14
mariadb_1     | 2023-05-06  2:38:51 0 [Note] InnoDB: Loading buffer pool(s) from /var/lib/mysql/ib_buffer_pool
mariadb_1     | 2023-05-06  2:38:51 0 [Note] Plugin 'FEEDBACK' is disabled.
mariadb_1     | 2023-05-06  2:38:51 0 [Warning] You need to use --log-bin to make --expire-logs-days or --binlog-expire-logs-seconds work.
mariadb_1     | 2023-05-06  2:38:51 0 [Note] Server socket created on IP: '0.0.0.0'.
mariadb_1     | 2023-05-06  2:38:51 0 [Note] Server socket created on IP: '::'.
mariadb_1     | 2023-05-06  2:38:51 0 [Note] InnoDB: Buffer pool(s) load completed at 230506  2:38:51
mariadb_1     | 2023-05-06  2:38:51 0 [Note] mysqld: ready for connections.
mariadb_1     | Version: '10.11.2-MariaDB-1:10.11.2+maria~ubu2204'  socket: '/run/mysqld/mysqld.sock'  port: 3306  mariadb.org binary distribution
photoprism_1  | started 230504 as root (amd64-prod)
photoprism_1  | Problems? Our Troubleshooting Checklists help you quickly diagnose and solve them:
photoprism_1  | https://docs.photoprism.app/getting-started/troubleshooting/
photoprism_1  | file umask....: "0002" (u=rwx,g=rwx,o=rx)
photoprism_1  | home directory: /photoprism
photoprism_1  | assets path...: /opt/photoprism/assets
photoprism_1  | storage path..: /photoprism/storage
photoprism_1  | config path...: default
photoprism_1  | cache path....: default
photoprism_1  | backup path...: /photoprism/storage/backups
photoprism_1  | import path...: /photoprism/import
photoprism_1  | originals path: /photoprism/originals
photoprism_1  | running as uid 0
photoprism_1  | /opt/photoprism/bin/photoprism start
photoprism_1  | time="2023-05-06T02:38:28Z" level=error msg="settings: could not create /photoprism/storage/config/settings.yml (open /photoprism/storage/config/settings.yml: no such file or directory)"
photoprism_1  | time="2023-05-06T02:38:30Z" level=info msg="config: plus extension loaded [1.832714486s]"
photoprism_1  | time="2023-05-06T02:38:55Z" level=info msg="Become a member today, support our mission and enjoy our member benefits! 💎"
photoprism_1  | time="2023-05-06T02:38:55Z" level=info msg="Visit https://www.photoprism.app/membership to learn more."
photoprism_1  | time="2023-05-06T02:38:56Z" level=info msg="migrate: running database migrations"
photoprism_1  | time="2023-05-06T02:38:56Z" level=info msg="migrate: 20221015-100000 successful [84.619102ms]"
photoprism_1  | time="2023-05-06T02:38:56Z" level=info msg="migrate: 20221015-100100 successful [33.279823ms]"
photoprism_1  | time="2023-05-06T02:39:35Z" level=info msg="migrate: 20211121-094727 successful [53.518642ms]"
photoprism_1  | time="2023-05-06T02:39:35Z" level=info msg="migrate: 20211124-120008 successful [22.036755ms]"
photoprism_1  | time="2023-05-06T02:39:35Z" level=info msg="migrate: 20220329-030000 successful [388.936342ms]"
photoprism_1  | time="2023-05-06T02:39:36Z" level=info msg="migrate: 20220329-040000 successful [544.680514ms]"
photoprism_1  | time="2023-05-06T02:39:36Z" level=info msg="migrate: 20220329-050000 successful [210.84188ms]"
photoprism_1  | time="2023-05-06T02:39:39Z" level=info msg="migrate: 20220329-060000 successful [3.333484272s]"
photoprism_1  | time="2023-05-06T02:39:40Z" level=info msg="migrate: 20220329-061000 successful [122.368629ms]"
photoprism_1  | time="2023-05-06T02:39:40Z" level=info msg="migrate: 20220329-070000 successful [288.847305ms]"
photoprism_1  | time="2023-05-06T02:39:40Z" level=info msg="migrate: 20220329-071000 successful [22.086826ms]"
photoprism_1  | time="2023-05-06T02:39:40Z" level=info msg="migrate: 20220329-080000 successful [377.674015ms]"
photoprism_1  | time="2023-05-06T02:39:41Z" level=info msg="migrate: 20220329-081000 successful [400.207157ms]"
photoprism_1  | time="2023-05-06T02:39:41Z" level=info msg="migrate: 20220329-083000 successful [22.120395ms]"
photoprism_1  | time="2023-05-06T02:39:41Z" level=info msg="migrate: 20220329-090000 successful [429.563891ms]"
photoprism_1  | time="2023-05-06T02:39:41Z" level=info msg="migrate: 20220329-091000 successful [337.46318ms]"
photoprism_1  | time="2023-05-06T02:39:41Z" level=info msg="migrate: 20220329-093000 successful [22.03221ms]"
photoprism_1  | time="2023-05-06T02:39:42Z" level=info msg="migrate: 20220421-200000 successful [366.921302ms]"
photoprism_1  | time="2023-05-06T02:39:43Z" level=info msg="migrate: 20220521-000001 successful [1.033616865s]"
photoprism_1  | time="2023-05-06T02:39:43Z" level=info msg="migrate: 20220521-000002 successful [288.4759ms]"
photoprism_1  | time="2023-05-06T02:39:44Z" level=info msg="migrate: 20220521-000003 successful [959.931381ms]"
photoprism_1  | time="2023-05-06T02:39:44Z" level=info msg="migrate: 20220927-000100 successful [241.373523ms]"
photoprism_1  | time="2023-05-06T02:39:44Z" level=info msg="migrate: 20221002-000100 successful [22.267268ms]"
photoprism_1  | time="2023-05-06T02:39:45Z" level=info msg="migrate: 20230102-000001 successful [211.138851ms]"
photoprism_1  | time="2023-05-06T02:39:45Z" level=info msg="migrate: 20230211-000001 successful [433.418949ms]"
photoprism_1  | time="2023-05-06T02:39:45Z" level=info msg="migrate: 20230309-000001 successful [48.557932ms]"
photoprism_1  | time="2023-05-06T02:39:45Z" level=info msg="migrate: 20230313-000001 successful [29.228561ms]"
photoprism_1  | time="2023-05-06T02:39:47Z" level=info msg="server: enabled gzip compression"
photoprism_1  | time="2023-05-06T02:39:47Z" level=info msg="webdav: shared /originals/"
photoprism_1  | time="2023-05-06T02:39:47Z" level=info msg="webdav: shared /import/"
photoprism_1  | time="2023-05-06T02:39:47Z" level=info msg="server: disabled tls"
photoprism_1  | time="2023-05-06T02:39:47Z" level=info msg="server: listening on 0.0.0.0:2342 [1.658392ms]"

**Comment by alexisLefebvre** [comment]
This is a duplicate of https://github.com/photoprism/photoprism/discussions/3377
